app/shared/models/git_repo.rb

```diff
@@ -106,32 +118,55 @@ def initialize(git_config: nil, local_folder: nil, provider_credential: nil, asy
         raise "unsupported credential type: #{provider_credential.type}"
       end
 
-      logger.debug("Adding task to setup repo #{self.git_config.git_url} at: #{local_folder}")
+      logger.debug("Adding task to setup repo #{git_config.git_url} at: #{local_folder}")
 
       setup_task = git_action_with_queue(ensure_block: proc { callback_block(async_start) }) do
-        logger.debug("Starting setup_repo #{self.git_config.git_url}".freeze)
+        logger.debug("Starting setup_repo #{git_config.git_url}".freeze)
         setup_repo
-        logger.debug("Done setup_repo #{self.git_config.git_url}".freeze)
```

Comment:
why change from `self.` to non `self.`

Reply:
We removed the `RedundantSelf` cop in the #462 PR.

Reply:
Nice, sorry I missed that :)